src/Microsoft.OData.Client/Microsoft.OData.Client.csproj

```diff
@@ -148,6 +148,10 @@
     <PackageReference Include="System.ComponentModel" Version="4.3.0" />
     <PackageReference Include="System.ComponentModel.Annotations" Version="4.7.0" />
   </ItemGroup>
+  
+  <ItemGroup>

```

Comment:
do we have the package to reference this?

Reply:
@xuzhg yes we do